modules/research-framework/research-service/src/main/java/org/apache/airavata/research/service/controller/LikesController.java

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
I believe it is better to move all the endpoints to the existing resource controller instead of creating a separate controller class